testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/securityapi/EESecurityAuthMechanismMultiConstraintsTestCase.java

```diff
@@ -161,7 +161,7 @@ static class ServerSetup extends AbstractElytronSetupTask {
 
         @Override
         protected ConfigurableElement[] getConfigurableElements() {
-            ConfigurableElement[] elements = new ConfigurableElement[3];
+            ConfigurableElement[] elements = new ConfigurableElement[2];
```

Comment:
This and the array in `SecurityAPITestCase` caused the tearDown method to fail because of the null element at the end. That messed with the configuration of the `EESecurityInjectionEnabled` test cases that ran afterwards and caused them to fail.